httpParser.hpp

```diff
@@ -0,0 +1,37 @@
+#ifndef WEBSERV_HTTPPARSER_HPP
```

Comment:
Why do you use `WEBSERV_HTTPPARSER_HPP` instead of `HTTPPARSER_HPP` as header guard?